packages/account/package.json

```diff
@@ -37,26 +37,26 @@
   "homepage": "https://github.com/ethereumjs/ethereumjs-vm/tree/master/packages/account#synopsis",
   "dependencies": {
     "ethereumjs-util": "^7.0.2",
-    "rlp": "^2.2.1",
+    "rlp": "^2.2.3",
     "safe-buffer": "^5.1.1"
   },
   "devDependencies": {
-    "@ethereumjs/config-nyc": "^1.1.0",
-    "@ethereumjs/config-prettier": "^1.1.0",
-    "@ethereumjs/config-tsc": "^1.1.0",
-    "@ethereumjs/config-tslint": "^1.1.0",
+    "@ethereumjs/config-nyc": "^1.1.1",
```

Comment:
For my understanding: this is just a cosmetic change, right? Or does this make a difference on CI caching or something?

Reply:
@holgerd77 Even though the version ranges result in the same version being used, it is a requirement for `lerna bootsrap --hoist` to work properly. See big chunk of warnings described in #730.

Reply:
or more explicitly: the repo must use the same exact versions so they get properly hoisted.